src/NugetReferenceResolver/NugetReferenceResolver.csproj

```diff
@@ -5,13 +5,13 @@
   <PropertyGroup>
     <Description>Resolves the dependency graph for a given set of project dependencies and computes different subsets of the dependency graph</Description>
     <VersionPrefix>1.0.1</VersionPrefix>
-    <TargetFrameworks>netstandard1.6;net451</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
     <OutputType>library</OutputType>
```

Comment:
Nit: as long as we're doing cleanup, we can remove this as it is the default value. This is true in a few other projects as well.